http4s/server/src/main/scala/endpoints4s/http4s/server/Endpoints.scala

```diff
@@ -374,6 +383,9 @@ trait EndpointsWithCustomErrors extends algebra.EndpointsWithCustomErrors with M
   def handleClientErrors(invalid: Invalid): http4s.Response[Effect] =
     clientErrorsResponse(invalidToClientErrors(invalid))
 
+  def handleClientErrorsEffect(invalid: Invalid): Effect[http4s.Response[Effect]] =
```

Comment:
We should document that this method is called when decoding a request failed. By the way, do we still call `handleClientErrors`? If not, we should update its documentation accordingly. Or, maybe better, we should probably only keep one method (the one that returns an `Effect[Response[Effect]]`), otherwise users may get confused: which method needs to be overridden?

I would be in favor of breaking the backward compatibility and to change the return type of `handleClientErrors` instead of introducing another operation. What do others think?

Reply:
`handleClientErrors` shouldn't be called anywhere internally anymore if i didn't mess up. I also agree with removing the pure handle functions since the rewrite to accomodate the effectful version on use site should be fairly minimal. Should I hold on the changes until the discussion played out or already switch it around?

Reply:
> Should I hold on the changes until the discussion played out or already switch it around?

Let’s do the switch already :)

Reply:
done

Reply:
I would rather keep the same name as before, `handleClientErrors`. Or do you think we should keep the suffix `Effect`? If that’s the case, we need to update the manual to use the new name.

Reply:
I agree it makes sense to keep the name, I was just too quick with my previous reiteration.